src/riverroute/RtmSpmd.F90

```diff
@@ -1,92 +1,64 @@
-
 module RtmSpmd
 
-!-----------------------------------------------------------------------
-!BOP
-!
-! !MODULE: RtmSpmd
-!
-! !DESCRIPTION:
-! RTM SPMD initialization
-!
-! !REVISION HISTORY:
-! Author: Mariana Vertenstein
-!
-!EOP
-!-----------------------------------------------------------------------
-  implicit none
-  private
-
-#include <mpif.h>  
-
-  save     ! This statement won't be needed once all compilers we support are compliant with FORTRAN-2008
-
-  ! Default settings valid even if there is no spmd 
-
-  logical, public :: masterproc      ! proc 0 logical for printing msgs
-  integer, public :: iam             ! processor number
-  integer, public :: npes            ! number of processors for rtm
-  integer, public :: mpicom_rof      ! communicator group for rtm
-  integer, public :: ROFID           ! mct compid
-  integer, public, parameter :: MASTERTASK=0 ! the value of iam which is assigned 
-                                             ! the masterproc duties
-
-  !
-  ! Public methods
-  !
-  public :: RtmSpmdInit                ! Initialization
-
-  !
-  ! Values from mpif.h that can be used
-  !
-  public :: MPI_INTEGER
-  public :: MPI_REAL8
-  public :: MPI_LOGICAL
-  public :: MPI_SUM
-  public :: MPI_MIN
-  public :: MPI_MAX
-  public :: MPI_LOR
-  public :: MPI_STATUS_SIZE
-  public :: MPI_ANY_SOURCE
-  public :: MPI_CHARACTER
-  public :: MPI_COMM_WORLD
-  public :: MPI_MAX_PROCESSOR_NAME
+   ! SPMD initialization
 
-contains
+   implicit none
+   private
+
+#include <mpif.h>
```

Comment:
I think that we should change this to 
  use mpi
It would need to be moved above the implicit none statement.

Reply:
Thanks. I've actually updated things so that 'use mpi' is now used in the necessary two routines rather than in RtmSpmd.F90.